app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
Using the ERD diagram, there is author_id as a foreign key.
this should be `belongs_to :author, class_name: 'User'`